core/src/main/scala/org/http4s/headers/Sec-WebSocket-Accept.scala

```diff
@@ -19,21 +19,28 @@ package headers
 
 import org.typelevel.ci._
 import org.http4s.internal.parsing.Rfc7230
+import java.util.Base64
 
-final case class `Sec-WebSocket-Accept`(accept: String)
+final class `Sec-WebSocket-Accept`(hashBytes: Array[Byte]) {
```

Comment:
Is the `Array` vs. a `ByteVector` for performance reasons?  It's odd for us to expose a mutable type.

Reply:
Yeah a public field was an oversight. the `Array[Byte]` is coming from the key hasher, but it makes more sense to convert or hide